### PR TITLE
TD-Migration: Fix the shell string split bug

### PR DESCRIPTION
**Pull request by LeiZhou-97**
Signed-off-by: LeiZhou <lei.zhou@intel.com>

### CODE REVIEW

**Review by ruomengh — APPROVED** [review]
LGTM

File: utils/td-migration/sourceTD.sh [hunk 1]
```diff
@@ -81,25 +81,25 @@ error() {
...
+        -bios ${OVMF} \
+        -chardev stdio,id=mux,mux=on,logfile=${LOG} \
+        -drive file=$(readlink -f "${GUEST_IMG}"),if=virtio,id=virtio-disk0,format=qcow2 \
+        -name process=lm-src,debug-threads=on \
+        -no-hpet -nodefaults \
+        -D /run/qemu-src.log -nographic -vga none \
+        -monitor unix:/tmp/qmp-sock-src,server,nowait \
+        -device virtio-serial,romfile= \
+        -device virtconsole,chardev=mux -serial chardev:mux -monitor chardev:mux \
+        -device virtio-net-pci,netdev=mynet0,mac=00:16:3E:68:00:10,romfile= \
```
Comment (kenplusplus) [inline utils/td-migration/sourceTD.sh:101]: Do we need hard code same MAC ?

Comment (LeiZhou-97) [reply utils/td-migration/sourceTD.sh]: Not needed. It's just a fake mac. 

Now remove it.

**Comment by LeiZhou-97** [comment]
Close it. I'll create a new PR for adding new parameters.

**Pull request closed without merging**